UnitySDK/Assets/ML-Agents/Examples/SharedAssets/Scripts/SettingsOverrides.cs

```diff
@@ -0,0 +1,52 @@
+using UnityEngine;
+using MLAgents;
+
+public class SettingsOverrides : MonoBehaviour
+{
+    // Original values
+    Vector3 m_OriginalGravity;
+    float m_OriginalFixedDeltaTime;
+    float m_OriginalMaximumDeltaTime;
+    int m_OriginalSolverIterations;
+    int m_OriginalSolverVelocityIterations;
+
+    public float gravityMultiplier = 1.0f;
+    public float fixedDeltaTime = .02f;
+    public float maximumDeltaTime = .33f;
+
+    [Header("Advanced physics settings")]
+    public int solverIterations = 6;
+    public int solverVelocityIterations = 1;
+
+    public void Awake()
+    {
+        // Save the original values
+        m_OriginalGravity = Physics.gravity;
+        Debug.Log($"Time.fixedDeltaTime = {Time.fixedDeltaTime}  Time.maximumDeltaTime={Time.maximumDeltaTime}");
+        m_OriginalFixedDeltaTime = Time.fixedDeltaTime;
+        m_OriginalMaximumDeltaTime = Time.maximumDeltaTime;
+        m_OriginalSolverIterations = Physics.defaultSolverIterations;
+        m_OriginalSolverVelocityIterations = Physics.defaultSolverVelocityIterations;
+
+        // Override
+        Physics.gravity *= gravityMultiplier;
+        Time.fixedDeltaTime = fixedDeltaTime;
```

Comment:
Most scenes only care about gravity, but Crawler and Walker override these deltaTime and solver iterations settings.